lib/inflector.php

```diff
@@ -458,4 +458,15 @@ public function ordinalize($number)
 	{
 		return $number . $this->ordinal($number);
 	}
+
+	/**
+	 * Multi-byte safe ucfirst
+	 *
+	 * @param $str
+	 * @return string
+	 */
+	private static function ucfirst($str)
```

Comment:
One last thing, please define the static method at the top of the class, after `static public function get($locale = self::DEFAULT_LOCALE)`, and use `static private` rather than `private static`. Thanks.